Code/GraphMol/FileParsers/CMLWriter.cpp

```diff
@@ -0,0 +1,209 @@
+//  Copyright (C) 2020 Eisuke Kawashima
+//
+//  Chemical Markup Language, CML, schema 3
+//  http://www.xml-cml.org/
+//  See
+//  http://www.xml-cml.org/convention/molecular
+//  http://www.xml-cml.org/schema/schema3/schema.xsd
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include "FileParsers.h"
+#include <fstream>
+#include <sstream>
+#include <boost/format.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/xml_parser.hpp>
+#include <RDGeneral/Invariant.h>
+
+#if __cplusplus >= 201402L && __has_cpp_attribute(fallthrough)
+#define FALLTHROUGH [[fallthrough]]
+#else
+#define FALLTHROUGH
+#endif
+
+namespace RDKit {
+
+namespace {
+boost::property_tree::ptree molToPTree(const ROMol& mol, int confId,
+                                       bool kekulize) {
+  if (!mol.getNumConformers()) {
+    BOOST_LOG(rdErrorLog)
+        << "Cannot write molecules with no conformers to CML format\n";
+    return {};
+  }
+
+  auto romol{mol};
+  auto& rwmol = static_cast<RWMol&>(romol);
+  if (kekulize) MolOps::Kekulize(rwmol);
```

Comment:
RDKit style guide is basically braces for everything.

```
if (kekulize) {
  MolOps::Kekulize(rwmol);
}
```

Reply:
Braces are added.